src/main/java/frc/robot/subsystems/tiltedelevator/TiltedElevatorSubsystem.java

```diff
@@ -218,7 +218,7 @@ public void periodic() {
         // else {
             // Set PID reference
             extensionPidController.setReference(
-                MathUtil.clamp(targetExtension, 0, EXTENSION_LIMIT),
+                MathUtil.clamp(targetExtension, 0, EXTENSION_LIMIT + Units.inchesToMeters(.75)),
```

Comment:
Is this not redundant to the soft limit? Clamping here is fine but maybe have two class variables `topLimit` and `botLimit` so that we can assure they are always set to the same value (in case we change one and forget to change the other).